components/table/test/table.vdiff.js

```diff
@@ -0,0 +1,1229 @@
+										<tr>
+											<td>Cell 2-A</td>
+											<td>Cell 2-B</td>
+											<td>Cell 2-C</td>
+										</tr>
+									</tbody>
+								</table>
+							`);
+							await expect(elem).to.be.golden();
+						});
+
+						it('no-header-no-tbody', async() => {
+							const elem = await createFixture(html`
+								<table class="d2l-table">
+									<tr>
+										<td>Cell 1-A</td>
+										<td>Cell 1-B</td>
+										<td>Cell 1-C</td>
+									</tr>
+									<tr>
+										<td>Cell 2-A</td>
+										<td>Cell 2-B</td>
+										<td>Cell 2-C</td>
+									</tr>
+								</table>
+							`);
+							await expect(elem).to.be.golden();
+						});
+
+						it('rowspan', async() => {
+							const elem = await createFixture(html`
+								<table class="d2l-table">
+									<thead>
+										<tr>
+											<th rowspan="2">Country</th>
+											<th colspan="3">Fruit</th>
+										</tr>
+										<tr>
+											<th>Apples</th>
+											<th>Bananas</th>
+											<th>Pears</th>
+										</tr>
+									</thead>
+									<tbody>
+										<tr>
+											<th>Canada</th>
+											<td>$1.29</td>
+											<td>$0.79</td>
+											<td>$2.41</td>
+										</tr>
+										<tr>
+											<th>Mexico</th>
+											<td>$0.59</td>
+											<td>$0.38</td>
+											<td>$1.99</td>
+										</tr>
+									</tbody>
+								</table>
+							`);
+							await expect(elem).to.be.golden();
+						});
+
+						it('footer', async() => {
+							const elem = await createFixture(html`
+								<table class="d2l-table">
+									<thead>
+										<tr>
+											<th>Header A</th>
+											<th>Header B</th>
+											<th>Header C</th>
+										</tr>
+									</thead>
+									<tbody>
+										<tr>
+											<td>Cell 1-A</td>
+											<td>Cell 1-B</td>
+											<td>Cell 1-C</td>
+										</tr>
+									</tbody>
+									<tfoot>
+										<tr>
+											<td>Footer 1-A</td>
+											<td>Footer 1-B</td>
+											<td>Footer 1-C</td>
+										</tr>
+									</tfoot>
+								</table>
+							`);
+							await expect(elem).to.be.golden();
+						});
+
+						it('selected-one-row', async() => {
+							const elem = await createFixture(html`
+								<table class="d2l-table">
+									<thead>
+										<tr>
+											<th>Header A</th>
+											<th>Header B</th>
+											<th>Header C</th>
+										</tr>
+									</thead>
+									<tbody>
+										<tr>
+											<td>Cell 1-A</td>
+											<td>Cell 1-B</td>
+											<td>Cell 1-C</td>
+										</tr>
+										<tr selected>
+											<td>Cell 2-A</td>
+											<td>Cell 2-B</td>
+											<td>Cell 2-C</td>
+										</tr>
+										<tr>
+											<td>Cell 3-A</td>
+											<td>Cell 3-B</td>
+											<td>Cell 3-C</td>
+										</tr>
+									</tbody>
+								</table>
+							`);
+							await expect(elem).to.be.golden();
+						});
+
+						it('selected-top-bottom', async() => {
```

Comment:
There's a lot of repetition between the fixtures, where only 1 or 2 things change between thing -- like in these "selected" tests. It was quickest/easiest to just copy these in for now, but there are definitely opportunities to build some functions that take parameters to create them.